www/js/metrics/metricsTypes.ts

```diff
@@ -2,14 +2,22 @@ import { LocalDt } from '../types/serverData';
 import { METRIC_LIST } from './MetricsTab';
 
 type MetricName = (typeof METRIC_LIST)[number];
+
 type LabelProps = { [k in `label_${string}`]?: number }; // label_<mode>, where <mode> could be anything
-export type DayOfMetricData = LabelProps & {
+export type DayOfServerMetricData = LabelProps & {
   ts: number;
   fmt_time: string;
   nUsers: number;
   local_dt: LocalDt;
 };
 
+type ModeProps = { [k in `mode_${string}`]?: number }; // mode_<mode>, where <mode> could be anything
+export type DayOfClientMetricData = ModeProps & {
+  date: string; // yyyy-mm-dd
+};
+
+export type DayOfMetricData = DayOfClientMetricData | DayOfServerMetricData;
```

Comment:
Yes. Although I changed things a little so instead of `mode_XXX` it is `mode_confirm_XXX` (or `purpose_confirm_XXX`, `primary_ble_sensed_mode_XXX`, etc)